### PR TITLE
e2e test fixes and sanity check for terminology modal

### PR DESCRIPTION
**Pull request by aburkleaux-amazon**
*Issue #, if available:*

*Description of changes:*
Several changes to address problems with e2e tests:

- Add documentation on how to run and debug the tests
- the e2e test is failing because the initial state of the video player is now paused.  Also, the location of the button changed. Update test for new behavior
- Automatically update the chrome driver to match the version of Chrome running in the test environment
- Enable some testing for the terminology modal that was disabled

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/e2e/requirements.txt [hunk 1]
```diff
@@ -3,5 +3,6 @@ requests==2.22.0
 #urllib3==1.26.5
 #jsonschema==2.6.0
 requests_aws4auth
+webdriver-manager
```
Comment (ianwow) [inline test/e2e/requirements.txt:6]: Unpinned requirements will generate a warning by python `safety`.
```
pip3 install safety
safety check aws-media-insights-content-localization/test/e2e/requirements.txt
```

File: test/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Media Insights Content Localization Testing - How To
+
+### Overview
```
Comment (ianwow) [inline test/README.md:3]: Our future selves will probably thank our current selves if we instruct users how to run end-to-end tests in a Docker container. That approach will help prevent the "works on my machine" bug response scenario.

Comment (ianwow) [reply test/README.md:3]: e.g. The pupeteer tests in CAS run in Docker:
https://github.com/awslabs/aws-content-analysis/blob/development/.github/workflows/pr-workflow.yml

**Review by ianwow — APPROVED** [review]
I recommend dockerizing the e2e script and pinning packages in requirements.txt.

File: test/README.md (hunk 2 above)
Comment (aburkleaux-amazon) [reply test/README.md:3]: I added issue #155 to track this idea.

File: test/e2e/requirements.txt (hunk 1 above)
Comment (aburkleaux-amazon) [reply test/e2e/requirements.txt]: I pinned the packages.  I think Dependabot, which is installed on this project, serves the same purpose as a python `safety` scan, though.

**Review by ianwow — APPROVED** [review]

**Pull request merged**